drafts/use-cases/1.1.security-considerations.md

```diff
@@ -1,29 +1,31 @@
 ### Security considerations
 
 #### JSON array root security issues
-Due to security issues with JSON arrays which can be exploited in some circumstances, 
+Due to security issues with JSON arrays which can be exploited in some circumstances,
 there may be a need to avoid pure arrays in favour of a JSON object.
 In the scenario above, server returned a single object, which won't cause issues.
-But it is possible to return multiple resources, resulting in an JSON array returned 
+But it is possible to return multiple resources, resulting in an JSON array returned
 (server may not use any JSON-LD context or it may be completely different from the one used by the client).
 There are several possibilities here. One of them would be to use a *@graph* notation, i.e.:
 
-```javascript
+```json
 {
     "@graph": [
         {
             "@id": "/api",
             "operations": [
                 {
-                    "@id": "/api/events",
-                    "title": "Events",
-                    "httpMethod": "GET",
+                    "@id": "/api#list-events",
+                    "@type": "hydra:Operation",
+                    "title": "List events",
+                    "method": "GET",
                     "returns": "hydra:Collection"
                 },
                 {
-                    "@id": "/api/events",
+                    "@id": "/api#add-event",
```

Comment:
Looking at the entrypoint use case, the `/api` resource is also part of the API's _data_. I would try to keep it explicitly separate. Similar to what you suggested, I'd go with a separate resource such as `/api/documentation` and then deep links to operations `/api/documentation#events/add-event`. This is similar to what @lanthaler implemented in the Console demo